src/FolderConfig.cpp

```diff
@@ -0,0 +1,54 @@
+#include "FolderConfig.h"
+#include <QFile>
+#include <QDebug>
+#include<QDir>
+
+namespace QtDBMigrationNS {
+FolderConfig::FolderConfig(const QString &path): AbstractConfig(path)
+{
+}
+
+bool FolderConfig::load()
+{
+    QDir dir(m_path);
+    QStringList nameFilter("*.sql");
+    QStringList sqlFiles = dir.entryList(nameFilter);
+    int totalFiles = sqlFiles.count();
+
+    if (!dir.exists() || totalFiles < 1) {
+        return false;
+    }
+
+    foreach(QString filename, sqlFiles) {
+        QString filePath = dir.filePath(filename);
+        QFile file(filePath);
+        if (!file.exists()) {
+            return false;
+        }
+
+        QFileInfo fileInfo(file.fileName());
+        QString fileName(fileInfo.baseName());
+        QRegExp rx("[0-9][0-9][0-9][0-9]");
+        if (!rx.exactMatch(fileName)) {
```

Comment:
For exact, I think you need to add `^` (start of line) and `$` (end of line) to regex.

Reply:
Yes it is true, added both symbols.